tests/QueryBuilderTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace CanalTP\NavitiaPhp\Tests;
+
+use CanalTP\NavitiaPhp\QueryBuilder;
+
+class QueryBuilderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetQueryWithNothing()
+    {
+        $queryBuilder = new QueryBuilder();
+
+        $query = $queryBuilder->getQuery();
+
+        $this->assertEquals('', $query);
+    }
+
```

Comment:
it would be nice to test the very specif call /coverage :
$queryBuilder->coverage('')